app.js

```diff
@@ -15,224 +15,224 @@
-  LOG_APPJS('Loading option specified.')
-  logic.bootstrapFile(argv.load)
-  isPersistence = true
+  LOG_APPJS('Loading option specified.');
+  logic.bootstrapFile(argv.load);
+  isPersistence = true;
 }
 
 if (process.env.NODE_ENV === 'test') {
-  argv.accounts = 'test/account-fixtures.json'
+  argv.accounts = 'test/account-fixtures.json';
 }
 
 /* account creation/loading based on presets given */
 if (argv.accounts) {
-  LOG_APPJS(`Bootstrapping from account fixture files: ${argv.accounts}`)
-  const accountsPath = argv.accounts
+  LOG_APPJS(`Bootstrapping from account fixture files: ${argv.accounts}`);
+  const accountsPath = argv.accounts;
   if (!fs.existsSync(accountsPath)) {
-    throw new Error('Account Path Invalid')
+    throw new Error('Account Path Invalid');
   }
-  const accounts = JSON.parse(fs.readFileSync(accountsPath, 'utf-8'))
-  wallet.loadAccounts(accounts)
+  const accounts = JSON.parse(fs.readFileSync(accountsPath, 'utf-8'));
+  wallet.loadAccounts(accounts);
 } else {
   /* Create Dummy Accounts */
   // create 10 wallets by default
-  wallet.createWallets(config.wallet.numAccounts)
+  wallet.createWallets(config.wallet.numAccounts);
 }
-wallet.printWallet()
+wallet.printWallet();
 
 // cleanup old folders
 if (fs.existsSync('./tmp')) {
-  LOG_APPJS(`Tmp folder found. Removing ${__dirname}/tmp`)
-  rimraf.sync(path.join(__dirname, '/tmp'))
-  LOG_APPJS(`${__dirname}/tmp removed`)
+  LOG_APPJS(`Tmp folder found. Removing ${__dirname}/tmp`);
+  rimraf.sync(path.join(__dirname, '/tmp'));
+  LOG_APPJS(`${__dirname}/tmp removed`);
 }
 
 if (!fs.existsSync('./tmp')) {
-  fs.mkdirSync('./tmp')
-  LOG_APPJS(`tmp folder created in ${__dirname}/tmp`)
+  fs.mkdirSync('./tmp');
+  LOG_APPJS(`tmp folder created in ${__dirname}/tmp`);
 }
 if (!fs.existsSync('./data')) {
-  fs.mkdirSync('./data')
-  fsp.mkdir('./data/save', 777, true, err => {
+  fs.mkdirSync('./data');
+  fsp.mkdir('./data/save', 777, true, (err) => {
     if (err) {
-      console.log(err)
+      console.log(err);
     } else {
-      LOG_APPJS('Directory created')
+      LOG_APPJS('Directory created');
     }
-  })
+  });
 }
 
 const wrapAsync = fn => (req, res, next) => {
-  Promise.resolve(fn(req, res, next)).catch(next)
-}
+  Promise.resolve(fn(req, res, next)).catch(next);
+};
 
 // cross region settings with Env
 if (process.env.NODE_ENV === 'dev') {
-  expressjs.use(cors())
-  LOG_APPJS('CORS Enabled.')
+  expressjs.use(cors());
+  LOG_APPJS('CORS Enabled.');
 }
 
 expressjs.get('/', (req, res) => {
-  res.status(200).send('Kaya RPC Server')
-})
+  res.status(200).send('Kaya RPC Server');
+});
 
 // Method handling logic for incoming POST request
 
 const handler = async (req, res) => {
-  const { body } = req
-  let data = {}
-  let result
-  let addr
-  LOG_APPJS(`Method specified: ${body.method}`)
+  const { body } = req;
+  let data = {};
+  let result;
+  let addr;
+  LOG_APPJS(`Method specified: ${body.method}`);
   switch (body.method) {
```

Comment:
I think it is better to wrap the entire `switch` statement in a `try...catch` block, rather than having one per-case.

Reply:
As discussed, we will refactor this portion together with protobuf implementation.